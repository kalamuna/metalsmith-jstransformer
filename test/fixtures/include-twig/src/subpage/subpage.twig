---
buttons:
  - title: Hello
  - title: World
  - title: How
  - title: Are
  - title: You
title: Subpage
---
<h1>{{ title }}</h1>
<div class="buttons">
  {% include '../components/button-group/_button-group.twig' %}
</div>
