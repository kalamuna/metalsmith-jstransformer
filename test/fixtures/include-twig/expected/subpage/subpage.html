<h1>Subpage</h1>
<div class="buttons">
    <ul>
          <li>
        <button>Hello</button>
      </li>
          <li>
        <button>World</button>
      </li>
          <li>
        <button>How</button>
      </li>
          <li>
        <button>Are</button>
      </li>
          <li>
        <button>You</button>
      </li>
      </ul>
</div>
